recipes/foxplainer/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "foxplainer" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 143e9bd95639d61a97b46190e42f2caafcd8f5353b6de6568e7759764ccd48e2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.8,<3.10.0
```

Comment:
Are you sure you need the upper bound on these things? What does python 3.10 break for you? Mostly it's better to leave things open, and to retroactively introduce pins if something breaks. This is something conda-forge achieves by patching repodata for existing packages. You don't need to worry about how it works right now, just know that it is possible, and that you don't need to pin too tightly.